build.gradle

```diff
@@ -24,16 +30,79 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
     antlr 'org.antlr:antlr4:4.5'
 
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    // using groovy 2.3 to avoid problems in eclipse
+    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
+    testCompile 'org.spockframework:spock-core:1.0-groovy-2.3'
+}
+
+//jar {
+//    manifest {
+//        attributes(
+//            'Built-Date': currentDate(),
+//            'Implementation-Version': version,
+//            'Implementation-Build': versioning.info.commit
+//        )
+//    }
+//}
+
+generateGrammarSource {
+    group "${project.name}"
+    description 'Generates the parser code based on the grammar'
+
+    outputDirectory = file("${outputDirectory}/org/asciidoctor/parser")
+}
+
+task pickAsciidocFile << {
```

Comment:
I personally don't think it is a good idea to add GUI stuff into the build file.